scaffolding-chef/lib/scaffolding.sh

```diff
@@ -164,6 +164,25 @@ token = "set_to_your_token"
 server_url = "set_to_your_url"
 EOF
   chmod 0640 "$pkg_prefix/default.toml"
+
+  if [ -d "$PLAN_CONTEXT/../data_bags" ]; then
+    _data_bag_path="$PLAN_CONTEXT/../data_bags"
+  else
+    if [ -d "$PLAN_CONTEXT/../../data_bags" ]; then
+      _data_bag_path="$PLAN_CONTEXT/../../data_bags"
+    else
+      if [ -d "$PLAN_CONTEXT/../../../data_bags" ]; then
+        _data_bag_path="$PLAN_CONTEXT/../../../data_bags"
+      else
+        echo "Cannot detect a data_bags directory"
```

Comment:
`build_line` is available for outputting lines of info.
I'm not sure it's functionally any different to echo but I imagine it at least makes it possible to customise where lines like this end up.